src/main/scala/rps/Main.scala

```diff
@@ -59,4 +54,16 @@ object WebServer extends App with RouterDerivationModule {
   object Draw
 }
 
-case class Response (userMove: Move, computerMove: Move, result: Result)
\ No newline at end of file
+case class Response (userMove: Move, computerMove: Move, result: Result)
+
+trait ID {
```

Comment:
separate ID model from the method used to generate it, just use 
`case class GameId(value: Int) extends AnyVal`
so scala will treat it as a Int at runtime, with no overhead!
https://docs.scala-lang.org/overviews/core/value-classes.html